tools/z_stack_splitter.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import click
+import numpy as np
+
+from MetaArray import MetaArray
+
+
+def split_z_stacks(arr, round_to=0.1, min_length=4):
```

Comment:
This is only used to split apart the scanning z-stack recordings you made, so maybe this goes away once we're done testing on saved data.